modules/machineset-creating-imds-options.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating_machinesets/creating-machineset-aws.adoc
+
+:_content-type: PROCEDURE
+[id="machineset-creating-imds-options_{context}"]
+= Configuring IMDS by using machine sets
+
+You can specify whether to require the use of IMDSv2 by adding or editing the value of `metadataServiceOptions.authentication` in your machine set YAML file.
```

Comment:
Maybe:
```suggestion
You can specify whether to require the use of IMDSv2 by adding or editing the value of `metadataServiceOptions.authentication` in the machine set YAML file for your compute machines.
```

Consider a mildly tangential note either here or in the concept with a link to how to do this with the AWS CLI for control plane machines.

Reply:
Great idea. Will add a link from where I talk about this in the concept as soon as I find one :slightly_smiling_face: